CMakeLists.txt

```diff
@@ -226,9 +227,7 @@ if(UNIX)
     ## flags for GCC
     # default to hidden symbols
     # ensure compatibility with older CPUs
-    # enable C++11 features
     add_definitions(-DLINUX_BUILD)
-    add_definitions(-D_GLIBCXX_USE_C99)
```

Comment:
Heads-up that this came from 7bbfd456d3, so we may be breaking compatibility with older systems here.